frontend/src/pages/team/Billing.vue

```diff
@@ -51,7 +51,32 @@
                     Something went wrong loading your subscription information, please try again.
                 </div>
             </div>
-            <EmptyState v-else-if="!isUnmanaged">
+            <EmptyState v-else-if="billingDisabledForTeam">
+                <template #img>
+                    <img src="../../images/empty-states/team-instances.png">
+                </template>
+                <template #header>Team Billing</template>
+                <template #message>
+                    <p>
+                        Your team does not require billing to be setup.
+                    </p>
+                </template>
+            </EmptyState>
+            <EmptyState v-else-if="isUnmanaged">
+                <template #img>
+                    <img src="../../images/empty-states/team-instances.png">
+                </template>
+                <template #header>Team Billing</template>
+                <template #message>
+                    <p>
+                        Your team billing cannot currently be managed from the dashboard.
+                    </p>
+                    <p>
+                        Please contact <a href="https://flowfuse.com/support/" class="underline" target="_blank">Support</a> for help.
+                    </p>
+                </template>
+            </EmptyState>
+            <EmptyState v-else>
                 <template #img>
                     <img src="../../images/empty-states/team-instances.png">
                 </template>
```

Comment:
For a lack of a better place to mention this, ill do it here.

Two things come to mind after creating a new freemium tier, upgrading to team tier, downgrading to freemium:
- not sure what to make out of my stripe customer portal billing info (could be development inconsistencies and the subscriptions i configured my local teams). What is a `small project`, `team plan and small project` and `team plan`? I only have a single team with properly configure billing details locally
![image](https://github.com/user-attachments/assets/7fc301c5-ab25-4cc1-868b-4812d52c6db7)
- billing wise, how can we apply the credit to the next invoice if the current team plan is free ?
![image](https://github.com/user-attachments/assets/12ce46a4-9da0-412c-853f-d6f53c137e08)

Reply:
> What is a small project, team plan and small project and team plan? I only have a single team with properly configure billing details locally

Those are the stripe product names. The naming in the test catalogue needs updating to match the more explicit and consistent naming we have in the production catalogue.

> billing wise, how can we apply the credit to the next invoice if the current team plan is free ?

The credit will get applied towards their $0 invoice... the credit will only get spent when they have a non-$0 invoice.